pymc3/tests/test_parallel_sampling.py

```diff
@@ -119,7 +123,8 @@ def test_abort():
 
 
 @pytest.mark.xfail(
-    reason="Possibly the same issue described in https://github.com/pymc-devs/pymc3/pull/4701"
+    condition=(IS_WINDOWS or IS_FLOAT32),
```

Comment:
Your fix might have changed the behavior of this test actually, as it is using NUTS for only one of two continuous variables

Reply:
Yeah, that's why I assumed we do not need a new test.

Reply:
I see, you're right. But it should not depend on windows or float32 though

Reply:
Let me push a fix, I think it's just a question of converting the start to the right float type